core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -655,7 +655,7 @@ protected function row($strPalette=null)
 				$ctrl = 'ctrl_preview_' . substr(md5($image), 0, 8);
 
 				$strPreview = '
-<div id="' . $ctrl . '" class="tl_edit_preview" data-original-width="' . $objFile->viewWidth . '" data-original-height="' . $objFile->viewHeight . '">
```

Comment:
why did you remove this?

Reply:
Because it is not needed anymore, see the changes in the core.js file